packages/core/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -48,9 +55,9 @@ export const ErrorPage = ({
   const navigate = useNavigate();
 
   return (
-    <Grid container className={classes.container}>
+    <Grid className={classes.container}>
```

Comment:
I believe that you are expected to have either the container or item prop on each Grid element, right?

Reply:
Giving `<Grid/>` a container prop was making unwanted scroll on both web and mobile. Now, I have added **container** prop but with it I have also added **spacing** prop with the value 0. It is working fine now even with container prop.